extras/catch_amalgamated.cpp

```diff
@@ -3550,7 +3550,7 @@ namespace Catch {
             // Extracts the actual name part of an enum instance
             // In other words, it returns the Blue part of Bikeshed::Colour::Blue
             StringRef extractInstanceName(StringRef enumInstance) {
-                // Find last occurence of ":"
```

Comment:
Neither of the `extras/catch_amalgamated.{h,c}pp` files should be touched outside of creating new release.